src/workspaces/WorkspacePolicies/WorkspacePolicies.ts

```diff
@@ -15,9 +14,7 @@ type WorkspacePoliciesProps = {
 
 export const WorkspacePolicies = (props: WorkspacePoliciesProps): ReactNode => {
   const policyDescriptions = getPolicyDescriptions(props.workspace);
-  const description = props.policiesLabel
-    ? props.policiesLabel
-    : `This workspace has the following ${pluralize('policy', policyDescriptions.length)}:`;
+  const description = props.policiesLabel ? props.policiesLabel : 'This workspace has the following:';
```

Comment:
Removed "policies" from the end of the label because we don't want to show it for GCP workspaces, and I don't think it's critical for Azure given that we show "Policies" above this for Azure workspaces.